app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Create Account</h2>
+
+<%= devise_error_messages! %>
+
+<%# View for Devise method inherited by Registrations controller. %>
+<%# Uses `resource` for consistency with Devise::Registrations#new %>
+<%= form_for(resource,
+             as: resource_name,
+             url: registrations_path(resource),
+             html: { class: 'pss-admin-form' }) do |f| %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email, autofocus: true %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password, autocomplete: "off" %>
+  <% if @validatable %>
+  <div><em>(<%= @minimum_password_length %> characters minimum)</em></div>
+  <% end %>
+
+  <%= f.label :password_confirmation %>
+  <%= f.password_field :password_confirmation, autocomplete: "off" %>
```

Comment:
I must have been thinking of the edit page. Thanks for catching this.  I've removed the password fields. I'm having trouble getting the controller to reject password fields in `#create` because it's hard to override Devise's functionality (you can add allowed fields but it's not clear how to subtract them).  I don't think it's too much of a problem because nobody's going to try to fill in that field outside of a form submission, but I could look into it later.